common/transport/amqp/src/amqp_message.ts

```diff
@@ -3,14 +3,22 @@
 
 'use strict';
 
+import { uuid_to_string as rheaUuidToString, string_to_uuid as rheaStringToUuid, types as rheaTypes, message as rheaMessage } from 'rhea';
 import { Message } from 'azure-iot-common';
-import * as amqp10 from 'amqp10';
+
+/* tslint:disable:variable-name */
+/* tslint:enable:variable-name: [true, "check-format", allow-leading-underscore", "ban-keywords", "allow-snake-case"]*/
 
 function encodeUuid(uuidString: string): any {
   const uuidRegEx = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/;
   let uuid;
   if (typeof uuidString === 'string' && uuidString.match(uuidRegEx)) {
-    uuid = (<any>amqp10).Type.uuid(uuidString);
+    //
+    // SERIALIZATION DEPENDENCY!!
```

Comment:
>SERIALIZATION DEPENDENCY!! [](start = 7, length = 26)

nit: that specific line does not help

Reply:
Removed here and everywhere.

---
In reply to: [209349954](https://github.com/Azure/azure-iot-sdk-node/pull/344#discussion_r209349954) [](ancestors = 209349954)